docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -601,7 +633,55 @@ For WCOSS_CRAY:
    ``valid_param_vals script``. In addition, various example configuration files can be
    found in the ``regional_workflow/tests/baseline_configs`` directory.
 
+.. _VXConfig:
+
+Configure METplus Verification Suite (Optional)
+--------------------------------------------------
+
+Users who want to use the METplus verification suite to test their forecasts need to load the appropriate modules and add additional information to their ``config.sh`` file. Other users may skip to the :ref:`next section <SetUpPythonEnv>`. 
+
+.. note::
+   METplus is preinstalled on `Level 1 <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__ systems. METplus *installation* is currently not supported for this release of the SRW App, but METplus *use* is supported on systems with a functioning METplus installation. For more information about METplus, see :numref:`Section %s <MetplusComponent>`.
+
+Once installed, METplus users must load the appropriate modules (changing the module location and MET version to correspond to their system's installation):
```

Comment:
Should a path to the modulefiles directory be provided here?

Reply:
Yeah, I was thinking that the files that would need to be updated would be the run_vx.local under regional_workflow/modulefiles/tasks/ for whatever machine they are using.

Reply:
So I should change the "module use" command to: "module use -a ufs-srweather-app/regional_workflow/modulefiles/tasks/<machine>/run_vx.local"? 
Otherwise I'm not totally clear what you're saying (sorry :/ ).

Reply:
No worries. Maybe I am making it too complicated. 

What I am trying to convey is that say there was a new version of MET installed on Cheyenne, they would need to go into regional_workflow/modulefiles/tasks/cheyenne/run_vx.local and change the module that is being loaded (currently hardcoded to be module load met/10.0.0). 

This may not be worth mentioning but it was what I thought you were trying to convey with the sentence "Once installed, METplus users must load the appropriate modules (changing the module location and MET version to correspond to their system's installation):".

Reply:
What you have listed there is in the run_vx.local file.

Reply:
Not sure you want another opinion at the table, but I think it would be good to provide the files (run_vx.local under regional_workflow/modulefiles/tasks) that would need to be updated with a new module if a user chose to change the version of MET (presumably to a newer version).